File: codegen/results/v002/lmstudio/qwen_qwen3.8-27b-4bit_nothink/20260829_165520/python/001_csv_to_json/solution.py
import csv
import json

def main():
    with open('input/data.csv', 'r') as f:
        reader = csv.DictReader(f)
        data = []
        for row in reader:
            obj = {
                "Name": row["Name"].strip(),
                "Age": int(row["Age"]),
                "Email": row["Email"].strip(),
                "Score": float(row["Score"])
            }
            data.append(obj)
    print(json.dumps(data))

if __name__ == "__main__":
    main()

File: codegen/results/v002/lmstudio/qwen_qwen3.8-27b-4bit_nothink/20260829_165520/typescript/001_csv_to_json/solution.ts
import * as fs from 'fs';

const csvContent = fs.readFileSync('input/data.csv', 'utf-8');
const lines = csvContent.split('\n').filter(line => line.trim() !== '');
const header = lines[0].split(',');
const dataRows = lines.slice(1);

const result = dataRows.map(row => {
  const cols = row.split(',');
  return {
    Name: cols[0].trim(),
    Age: parseInt(cols[1].trim(), 10),
    Email: cols[2].trim(),
    Score: parseFloat(cols[3].trim())
  };
});

console.log(JSON.stringify(result, null, 2));